README.md

```diff
@@ -999,15 +1159,15 @@ For the identifier first flow,
 
 ### OpenID Connect
 
-Options for the [OpenID Connect](http://developer.okta.com/docs/api/resources/oidc.html) authentication flow. This flow is required for social authentication, and requires OAuth 2.0 client registration with Okta. For instructions, see [Social Authentication](http://developer.okta.com/docs/api/resources/social_authentication.html).
+Options for the [OAuth][] [OpenID Connect](http://developer.okta.com/docs/api/resources/oidc.html) authentication flow. This flow is required for social authentication, and requires OAuth 2.0 client registration with Okta. For instructions, see [Social Authentication](http://developer.okta.com/docs/api/resources/social_authentication.html).
 
 - **clientId:** Client Id pre-registered with Okta for the OIDC authentication flow
 
     ```javascript
     clientId: 'GHtf9iJdr60A9IYrR0jw'
     ```
 
-- **redirectUri:** The url that is redirected to when using `authParams.display:page`. This must be pre-registered as part of client registration. If no `redirectUri` is provided, defaults to the current origin.
+- **redirectUri:** The [callback][] url that is redirected to when using `authParams.display:page`. This must be pre-registered as part of client registration. If no `redirectUri` is provided, defaults to the current origin.
```

Comment:
Is the reference to authParams.display:page how we want to mention this?   That param hasn't been mentioned before, and really isn't defined well on the auth-js README.

Reply:
I just added the `[callback][]` part, I had not seen this before. Anyways, this information is not accurate. We no longer base redirect behavior on `authParams.display` as of version `5.0`.